android-components/components/concept/engine/src/main/java/mozilla/components/concept/engine/translate/TranslationOperation.kt

```diff
@@ -17,4 +17,10 @@ enum class TranslationOperation {
      * A translated page should be restored.
      */
     RESTORE,
+
+    /**
+     * Fetch the languages that the translation engine supports to populate the to and from
+     * dropdowns.
```

Comment:
nit: Maybe we shouldn't mention an specific UI use case for this constant :)

Reply:
For sure, updated! 😅